stream-compositions/services/payment-order-composition-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,61 @@
+server.port: 9005
+
+spring:
+  activemq:
+    broker-url: tcp://localhost:61616
+  zipkin:
+    enabled: false
+  main:
+    allow-bean-definition-overriding: true
+  codec:
+    max-in-memory-size: 10MB
+sso:
+  jwt:
+    internal:
+      type: signed
+      signature:
+        algorithm: HS256
+        key:
+          type: ENV
+          value: SIG_SECRET_KEY
+backbase:
+  activemq:
+    enabled: true
+  communication:
+    http:
+      client-secret: bb-secret
+      client-id: bb-client
+      access-token-uri: http://localhost:8181/oidc-token-converter-service/oauth/token
+  stream:
+    dbs:
+      user-manager-base-url: http://localhost:8086/user-manager
+      access-control-base-url: http://localhost:8086/access-control
+      arrangement-manager-base-url: http://localhost:8082/arrangement-manager
+      transaction-manager-base-url: http://localhost:8083/transaction-manager
+      payment-order-service-base-url: http://localhost:8090/payment-order-service
+    compositions:
+      paymentorder:
+        defaultStartOffsetInDays: 30
+        integration-base-url: http://localhost:7004
+        events:
+          enableCompleted: true
+          enableFailed: true
+
+eureka:
+  instance:
+    metadata-map:
+      public: true
+      role: live
+  client:
+    serviceUrl:
+      defaultZone: http://localhost:8080/registry/eureka/
+
+bootstrap:
+  enabled: false
+  # This is just na example. Replace arrangementId from your project database reference
```

Comment:
Can be removed or the example has to be added?

Reply:
removed